include/cantera/thermo/Elements.h

```diff
@@ -10,6 +10,8 @@
 
 #include "cantera/base/ct_defs.h"
 
+using namespace std;
```

Comment:
`using namespace std` should *never* appear in a header file, since that forces any user code which may indirectly include this header to also be using the namespace.